scripts/build/builders/efr32.py

```diff
@@ -68,6 +68,8 @@ def FlashBundleName(self):
     def BuildRoot(self, root):
         if self == Efr32App.UNIT_TEST:
             return os.path.join(root, 'src', 'test_driver', 'efr32')
+        if self == Efr32App.LIGHT:
+            return os.path.join(root, 'examples', self.ExampleName(), 'silabs/efr32')
```

Comment:
This should work for now. Please give an ETA for when the move would be done for other examples

Reply:
As discussed the ETA for the other apps is 14 dec